tendrl/gluster_integration/sds_sync/brick_utilization.py

```diff
@@ -0,0 +1,155 @@
+import os
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+ONE_GB_BYTES = 1073741824.0
+
+
+def _getMountPoint(path):
+    mount = os.path.realpath(path)
+    while not os.path.ismount(mount):
+        mount = os.path.dirname(mount)
+    return mount
+
+
+def _parseProcMounts(filter=True):
```

Comment:
current func name is not according to PEP8, change to small_case, same with other function names in this file